rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormProjectDataProvider.groovy

```diff
@@ -33,7 +33,10 @@ class GormProjectDataProvider implements RundeckProjectDataProvider {
     }
 
     Long create(RdProject data) throws DataAccessException {
-        Project project = new Project(description: data.getDescription(), name: data.getName())
+        Project project = new Project(
+                name: data.getName(),
+                description: data.getDescription(), 
+                state: data.getState())
```

Comment:
will we create a project in disabled state? should we just use enabled state as default here?

Reply:
This point is just data passing, so the default is indeed set by the caller who provides the RdProject object.